.github/CONTRIBUTING.md

```diff
@@ -13,22 +13,24 @@ To ensure that all developers follow the same guidelines for development, we hav
 - Clone this repository to your Go workspace:
 
 ```sh
-# Make sure your go paths are set if they aren't already
-export GOPATH=$HOME/go
-export PATH=$PATH:$GOPATH/bin
+# Make sure you are running go 1.11 or later
+# if you plan to clone into your current GOPATH then set the environment variable GO111MODULE=on
+# this will tell go to use the new modules support
 
 # Clone the project
-git clone git@github.com:target/flottbot.git $GOPATH/src/github.com/target/flottbot
+git clone git@github.com:target/flottbot.git somepath/src/github.com/target/flottbot
 ```
 
-- Build the project dependencies using [dep](https://github.com/golang/dep):
+- Build the project:
 
 ```sh
 # Change into the project directory
-cd $GOPATH/src/github.com/target/flottbot
+cd somepath/src/github.com/target/flottbot
```

Comment:
Shouldn't this just be `cd somepath/flottbot`?

Reply:
Good suggestion, I have updated, thanks.